src/nuklear_tree.c

```diff
@@ -49,14 +49,22 @@ nk_tree_state_base(struct nk_context *ctx, enum nk_tree_type type,
     widget_state = nk_widget(&header, ctx);
     if (type == NK_TREE_TAB) {
         const struct nk_style_item *background = &style->tab.background;
-        if (background->type == NK_STYLE_ITEM_IMAGE) {
-            nk_draw_image(out, header, &background->data.image, nk_white);
-            text.background = nk_rgba(0,0,0,0);
-        } else {
-            text.background = background->data.color;
-            nk_fill_rect(out, header, 0, style->tab.border_color);
-            nk_fill_rect(out, nk_shrink_rect(header, style->tab.border),
-                style->tab.rounding, background->data.color);
+
+        switch(background->type) {
+            case NK_STYLE_ITEM_IMAGE:
+                text.background = nk_rgba(0, 0, 0, 0);
+                nk_draw_image(out, header, &background->data.image, nk_white);
+                break;
+            case NK_STYLE_ITEM_9SLICE:
+                text.background = nk_rgba(0, 0, 0, 0);
+                nk_draw_9slice(out, header, &background->data.slice, nk_white);
+                break;
+            case NK_STYLE_ITEM_COLOR:
+                text.background = background->data.color;
+                nk_fill_rect(out, header, 0, style->tab.border_color);
+                nk_fill_rect(out, nk_shrink_rect(header, style->tab.border),
```

Comment:
On some places in the patch long lines are wrapped and on some not :wink:. I'll merge it anyway because `nuklear.h` historically didn't wrap lines much/at_all, but I'm just pointing it out as I find any inconsistency interesting.

Reply:
Do you have a preference as to how many characters long a line should be? I tried to adhere to the surrounding coding style as much as I could.

Reply:
I do (the good old 80 chars rule), but I'm convinced I shouldn't dictate it for this project. Adherence to the general coding style in `nuklear.h` is what counts. The only thing is, that I'd maybe slightly prefer if PRs themselves were consistent on their own :wink:.

Reply:
It might help to write up a style guide for future contributors, if it's not too much work. Just something simple like "use x naming scheme, keep lines under y characters long, make sure to use asserts for z".

Reply:
Yeah, that makes sense - on the other hand as I wrote, it's not a requirement, just an observation which won't affect whether to merge or not. So we might mention that as "preferred", but not a requirement in https://github.com/Immediate-Mode-UI/Nuklear/#reviewers-guide :wink:. Feel free to make a PR to formulate how you'd imagine that.